apps/server/src/modules/board/service/content-element.service.ts

```diff
@@ -41,6 +42,12 @@ export class ContentElementService {
 	}
 
 	async create(parent: Card | SubmissionItem, type: ContentElementType): Promise<AnyContentElementDo> {
+		if (type === 'drawing') {
+			const drawing = parent.children.find((child) => child instanceof DrawingElement);
```

Comment:
why is it still a problem to have multiple drawings? I was hoping this could just be removed.

Reply:
It remained because we've got business requirement that there should be one drawing tool per one card, so we have to check it.